modules/tinymce/src/core/test/ts/browser/selection/SimpleTableModelTest.ts

```diff
@@ -1,136 +1,132 @@
-import { Assertions, Chain, GeneralSteps, Logger, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { Fun, Result } from '@ephox/katamari';
+import { Assertions } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
 import { Hierarchy, Html, SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+
 import * as SimpleTableModel from 'tinymce/core/selection/SimpleTableModel';
 
-UnitTest.asynctest('browser.tinymce.core.selection.SimpleTableModel', (success, failure) => {
+describe('browser.tinymce.core.selection.SimpleTableModel', () => {
+  const fromDom = (html: string) => SimpleTableModel.fromDom(SugarElement.fromHtml(html));
 
-  const cFromDom = (html: string) => {
-    return Chain.injectThunked(() => {
-      return SimpleTableModel.fromDom(SugarElement.fromHtml(html));
-    });
+  const fromDomSubSection = (html: string, startPath: number[], endPath: number[]) => {
+    const tableElm = SugarElement.fromHtml<HTMLTableElement>(html);
+    const startElm = Hierarchy.follow(tableElm, startPath).getOrDie();
+    const endElm = Hierarchy.follow(tableElm, endPath).getOrDie();
+    return SimpleTableModel.subsection(SimpleTableModel.fromDom(tableElm), startElm, endElm).getOrDie('Failed to get the subsection');
   };
 
-  const cFromDomSubSection = (html: string, startPath: number[], endPath: number[]) => {
-    return Chain.binder((_) => {
-      const tableElm = SugarElement.fromHtml<HTMLTableElement>(html);
-      const startElm = Hierarchy.follow(tableElm, startPath).getOrDie();
-      const endElm = Hierarchy.follow(tableElm, endPath).getOrDie();
-      return SimpleTableModel.subsection(SimpleTableModel.fromDom(tableElm), startElm, endElm).fold(
-        Fun.constant(Result.error('Failed to get the subsection')),
-        Result.value
-      );
-    });
+  const assertWidth = (tableModel: SimpleTableModel.TableModel, expectedWidth: number) => {
+    assert.equal(tableModel.width, expectedWidth, 'Should be expected width');
   };
 
-  const cAssertWidth = (expectedWidth: number) => {
-    return Chain.op((tableModel: SimpleTableModel.TableModel) => {
-      Assertions.assertEq('Should be expected width', expectedWidth, tableModel.width);
-    });
+  const assertHeight = (tableModel: SimpleTableModel.TableModel, expectedWidth: number) => {
```

Comment:
Should this be height not width?

```suggestion
  const assertHeight = (tableModel: SimpleTableModel.TableModel, expectedHeight: number) => {
```

Reply:
Oof, this was a copy/paste fail. Thanks for the catch!